lib/clusterManager.js

```diff
@@ -187,7 +192,63 @@ if (cluster.isMaster) {
     this.overrideFileConfig = override;
   }
 
-   ClusterManager.prototype.notifyWorkersForAddingPlugin = function(resultHandler, indexInCluster) {
+  /* Having gotten here, we are already at the cluster, but it's safer to validate anyway
+    i. e. "if (this.storage) ..." since only the master cluster has storage */
+  ClusterManager.prototype.getStorage = function(resultHandler) {
+    if (this.storage) {
```

Comment:
I think I'll leave it the way I have it now for the sake of code clarity (and consistency, since we have this simple to read way nearly everywhere so we'd want to go back and change all those occurrences as well)